leapp/libraries/stdlib/__init__.py

```diff
@@ -165,15 +165,17 @@ def run(args, split=False, callback_raw=_console_logging_handler, callback_lineb
     :return: {'stdout' : stdout, 'stderr': stderr, 'signal': signal, 'exit_code': exit_code, 'pid': pid}
     :rtype: dict
     """
-    api.current_logger().debug('External command is started: [%s]', ' '.join(args))
+    if not args:
+        raise ValueError('Command to call is missing.')
+    api.current_logger().debug('External command has started: {0}'.format(str(args)))
```

Comment:
```suggestion
    api.current_logger().debug('External command has started: {0}'.format(' '.join(args)))
```

same goes for other messages in this PR